### ISSUE TITLE
[FEATURE REQUEST] Add another list to detect scam channel names

### ISSUE THREAD

**Issue by KendallDoesCoding** [ThioJoe/YT-Spam-Domains-List#44]
@ThioJoe I think it should auto detect channel names like

"CLICK MY CHANNEL"
"GO TO MY CHANNEL"
"FREE ROBUX"

And there's a whole list.

This would also fix, https://github.com/ThioJoe/YT-Spammer-Purge/issues/327 and other issues. I see lots of comments with channel names like stated above.

**Comment by ayjemabo** [comment]
Yeah @KendallDoesCoding because maybe like "Logan" you get hacked

**Comment by Sarah1390** [comment]
I was checking for safe ways to pull the domains without going to the actual site itself. These links are all tokenized and even further hidden by using URL shortening when I was able to get to the end of the URL where it contained the website re-directed to. If anything I would suspect you would have to write separate code in a separate program to scan the links  used by youtube channels for malicious sites. I know that McAfee has something similar with their WebAdvisor but to be put to use in conjuction with comments is way beyond the scope  and just easier to filter out the common phrases

**Comment by KendallDoesCoding** [comment]
> I was checking for safe ways to pull the domains without going to the actual site itself. These links are all tokenized and even further hidden by using URL shortening when I was able to get to the end of the URL where it contained the website re-directed to. If anything I would suspect you would have to write separate code in a separate program to scan the links used by youtube channels for malicious sites. I know that McAfee has something similar with their WebAdvisor but to be put to use in conjuction with comments is way beyond the scope and just easier to filter out the common phrases

Yes, this may need to be done too. He already has a system in place to scan scam links/urls posted in comments, but not scammers who say GO TO MY CHANNEL and click this link, but even if he implements #44, the spam commenters will go and nobody will know that comment even existed.

**Comment by ThioJoe** [comment]
Moving this to a discussion. I will point out that the program does indeed check for stuff like mentioned, it just wasn't working correctly at the time
